.github/workflows/humble-debian-build.yml

```
@@ -0,0 +1,33 @@
+name: Humble debian Build
```

```suggestion
name: Debian Humble Build
```
This'd make it easier to stand out as a Debian build from the rest of the build variants